js/languagePickerModel.js

```diff
@@ -102,7 +104,7 @@ define([
       if (!this.get('_restoreStateOnLanguageChange')) {
         return;
       }
-      this.listenToOnce(Adapt, 'menuView:ready', this.restoreLocation);
+      this.listenTo(Adapt, 'pageView:ready menuView:ready', this.restoreLocation);
```

Comment:
```suggestion
      this.listenTo(Adapt, 'contentObjectView:ready', this.restoreLocation);
```
https://github.com/adaptlearning/adapt_framework/blob/master/src/core/js/views/contentObjectView.js#L71